### DISCUSSION TITLE
System.TypeLoadException when compiling html

### DISCUSSION THREAD

**Discussion by DrHeinzDoofenshmirtz** [Tyrrrz/MiniRazor#16] [Q&A]
Hi.

I get the follow error when I try to do a `Razor.Compile` in an Azure Function App.

`System.TypeLoadException: 'Method 'CreateAnalyzerDriver' in type 'Microsoft.CodeAnalysis.CSharp.CSharpCompilation' from assembly 'Microsoft.CodeAnalysis.CSharp, Version=3.8.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' does not have an implementation.'`

This is how I do the compile:
`var template = Razor.Compile(html, alc);`

I am having a bunch of errors trying to execute the compilation from an Azure Function App.
Before this, I was having issues because Function Apps create a special `bin` sub-folder when building the project, and I had to add the following line to the `.csproj` file in order for it to work.
`<_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>`

I don't know why I get the error, and I can't seem to find a solution online.
If it helps, I am using MiniRazor inside a class library that the Function App is depending on.

I hope someone can help.

**Comment by Tyrrrz** [comment]
Could this have any useful information? https://stackoverflow.com/questions/58634755/method-commoncreatearraytypesymbol-does-not-have-an-implementation

Never used function apps, so can't advise from experience
